tests/installation/partitioning_olddisk.pm

```diff
@@ -0,0 +1,96 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary:  Install on old disk
+# Maintainer: Joyce Na <jna@suse.de>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use version_utils qw(is_storage_ng is_sle);
+use testapi;
+
+sub run() {
+    send_key $cmd{expertpartitioner};
+    #assert_screen 'expert-partitioner-setup';
+    if (is_storage_ng && is_sle('15+')) {
+        # start with preconfigured partitions
+        send_key 'down';
+        send_key 'ret';
+    }
+    assert_screen 'expert-partitioner-setup';
+    sleep 3;
+    # Rescan devices
+    if (is_storage_ng && is_sle('15+')) {
```

Comment:
I believe we can leave just `is_storage_ng` condition here.

Reply:
Updated,removed _is_sle('15+')_